packages/components/src/Actions/Actions.component.js

```diff
@@ -85,9 +108,13 @@ Actions.propTypes = {
 		PropTypes.oneOfType([
 			PropTypes.shape(Action.propTypes),
 			PropTypes.shape({
-				displayMode: PropTypes.oneOf([TYPE_DROPDOWN]),
+				displayMode: TYPE_DROPDOWN,
```

Comment:
error: shape is waiting key / proptypes.
proptypes is should be PropTypes.string or oneOf as before.